yarn-project/pxe/src/note_processor/produce_note_dao.ts

```diff
@@ -150,10 +196,53 @@ export async function produceNoteDaos(
           txHash,
           noteHashes,
           dataStartIndexForTx,
+          unencryptedLogs,
         );
       }
+    } else if ((e as any).message.includes('failed to solve blackbox function: embedded_curve_add')) {
```

Comment:
This code is basically a duplicate of the code above but I would not bother with some larger refactor now as it's a temporary solution.